src/components/EmojiSuggestions.tsx

```diff
@@ -46,30 +46,33 @@ function EmojiSuggestions({emojis, onSelect, prefix, isEmojiPickerLarge, preferr
     /**
      * Render an emoji suggestion menu item component.
      */
-    const renderSuggestionMenuItem = (item: SimpleEmoji): ReactElement => {
-        const styledTextArray = getStyledTextArray(item.name, prefix);
+    const renderSuggestionMenuItem = useCallback(
+        (item: SimpleEmoji): ReactElement => {
+            const styledTextArray = getStyledTextArray(item.name, prefix);
 
-        return (
-            <View style={styles.autoCompleteSuggestionContainer}>
-                <Text style={styles.emojiSuggestionsEmoji}>{EmojiUtils.getEmojiCodeWithSkinColor(item, preferredSkinToneIndex)}</Text>
-                <Text
-                    numberOfLines={2}
-                    style={styles.emojiSuggestionsText}
-                >
-                    :
-                    {styledTextArray.map(({text, isColored}) => (
-                        <Text
-                            key={`${text}+${isColored}`}
-                            style={StyleUtils.getColoredBackgroundStyle(isColored)}
-                        >
-                            {text}
-                        </Text>
-                    ))}
-                    :
-                </Text>
-            </View>
-        );
-    };
+            return (
+                <View style={styles.autoCompleteSuggestionContainer}>
+                    <Text style={styles.emojiSuggestionsEmoji}>{EmojiUtils.getEmojiCodeWithSkinColor(item, preferredSkinToneIndex)}</Text>
+                    <Text
+                        numberOfLines={2}
+                        style={styles.emojiSuggestionsText}
+                    >
+                        :
+                        {styledTextArray.map(({text, isColored}) => (
+                            <Text
+                                key={`${text}+${isColored}`}
+                                style={StyleUtils.getColoredBackgroundStyle(isColored)}
+                            >
+                                {text}
+                            </Text>
+                        ))}
+                        :
+                    </Text>
+                </View>
+            );
+        },
+        [styles, prefix, preferredSkinToneIndex],
```

Comment:
The styles object is quite large. Do you think that matters when including it in a dependency array? Would it be better to include all the used styles in the dependency array separately?